docs/packages/markdown/loader.js

```diff
@@ -99,12 +123,13 @@ module.exports = async function demoLoader() {
         moduleID.replace(/\//g, path.sep),
       );
       this.addDependency(moduleFilepath);
-      demos[demoName] = {
-        module: moduleID,
-        raw: await fs.readFile(moduleFilepath, { encoding: 'utf-8' }),
-      };
+      const raw = await fs.readFile(moduleFilepath, { encoding: 'utf-8' });
```

Comment:
@Janpot Why this change?

Reply:
I'm assigning the demo source to a variable so that I can analyse it later for import statements. That way I don't have to read it twice from the file system.